### PR TITLE
Check for old 'enterprise' module

### PR DESCRIPTION
**Pull request by yurabakhtin**
Issue: https://github.com/humhub/updater/issues/31

### CODE REVIEW

File: libs/UpdatePackage.php [hunk 1]
```diff
@@ -157,6 +158,31 @@ public function checkPhpVersion()
...
+        // Conditions when modules should be restricted:
+        $restrictedModules = [
+            'enterprise' => ['HumHubVersion', '>=', '1.10']
+        ];
+
+        foreach ($restrictedModules as $moduleId => $restriction) {
+            if (Yii::$app->getModule($moduleId) &&
+                $restriction[0] === 'HumHubVersion' &&
+                version_compare($this->getNewConfigValue('version'), $restriction[2], $restriction[1])) {
+                return Yii::t('UpdaterModule.base', 'This version is not supported with the legecy Enterprise Edition. Please contact support! {email}', [
```
Comment (dreua) [inline libs/UpdatePackage.php:177]: typo: legacy

Comment (yurabakhtin) [reply libs/UpdatePackage.php]: @dreua Thanks! Fixed in https://github.com/humhub/updater/pull/32/commits/827d4206484d8575adb1d5f795d10c090a3ae75d.

File: libs/UpdatePackage.php [hunk 2]
```diff
@@ -157,6 +158,31 @@ public function checkPhpVersion()
...
+    /**
+     * Check for restricted modules
+     *
+     * @return string|true TRUE - if no restriction, String - error message
+     */
+    public function checkRestrictedModules()
+    {
+        // Conditions when modules should be restricted:
+        $restrictedModules = [
+            'enterprise' => ['HumHubVersion', '>=', '1.10']
```
Comment (luke-) [inline libs/UpdatePackage.php:170]: Wouldn't it be good to include the error message in the array? In case further restrictions are added later?

**Review by luke- — CHANGES_REQUESTED** [review]

File: libs/UpdatePackage.php (hunk 2 above)
Comment (yurabakhtin) [reply libs/UpdatePackage.php]: @luke- Done in commit https://github.com/humhub/updater/pull/32/commits/3e7d5e0b6b9c0b0c97efbefe28edc5be6c02c080.

**Review by luke- — APPROVED** [review]

**Pull request merged**